android/library/src/main/java/io/logto/android/Logto.kt

```diff
@@ -67,8 +61,82 @@ class Logto(
         } ?: onComplete(Error("You are not signed in!"))
     }
 
-    private fun clearCredential() {
-        credentialStorage?.clearCredential()
-        credentialCache = null
+    fun getAccessToken(
+        block: (error: Error?, accessToken: String?) -> Unit
+    ) {
+        if (credential == null) {
+            block(Error("Not Sign In!"), null)
+            return
+        }
+
+        if (System.currentTimeMillis() < accessTokenExpiresAt) {
+            block(null, credential?.accessToken)
+            return
+        }
+
+        refreshCredential { error, credential ->
+            block(error, credential?.accessToken)
+        }
+    }
+
+    fun refreshCredential(
```

Comment:
nit: Per offline discussion, update the `credential` namespace as well as the func name. Thx

Reply:
TBD: [186](https://linear.app/silverhand/issue/LOG-186/follow-naming-conventions-with-js-sdk)